front/src/modules/simulationResult/components/SimulationResultsMap/RenderItinerary.tsx

```diff
@@ -38,18 +38,18 @@ export default function RenderItinerary({
 
   return (
     <Source type="geojson" data={geojsonPath}>
-      <Marker longitude={originLongitude} latitude={originLatitude} anchor="bottom">
+      <Marker longitude={originLongitude} latitude={originLatitude} anchor="bottom" offset={[0, 8]}>
```

Comment:
Note, I've duplicated the offset 3 times because there's no real reason why these all need to match up (ie, they are the same "by chance"). If you prefer to store the offset in a `const` or something, that's fine by me as well.

Reply:
Good work! Since offset [0, 8]  is used to adjust visual alignment with the path, it might seem logical to extract it in a const like `const markerAlignmentOffset = [0, 8];`. However, if these offsets could diverge in the future (e.g., specific adjustments for start vs. intermediate points), keeping them separate as they are might actually make sense.

Reply:
So, what is your preference?

Reply:
store the offset in a const

Reply:
Done. Had to explicitly make the const a `[number, number]` other TypeScript treats it as a `number[]`.